internal/nonkube/client/fs/site_handler.go

```diff
@@ -34,6 +38,79 @@ func (s *SiteHandler) Add(resource v2alpha1.Site) error {
 	return nil
 }
 
-func (s *SiteHandler) Get(name string, opts GetOptions) (*v2alpha1.Site, error) { return nil, nil }
-func (s *SiteHandler) Delete(name string) error                                 { return nil }
-func (s *SiteHandler) List() ([]*v2alpha1.Site, error)                          { return nil, nil }
+func (s *SiteHandler) Get(name string, opts GetOptions) (*v2alpha1.Site, error) {
+	var context v2alpha1.Site
+	fileName := name + ".yaml"
+
+	if opts.RuntimeFirst == true {
+		// First read from runtime directory, where output is found after bootstrap
+		// has run.  If no runtime sites try and display configured sites
+		err, file := s.ReadFile(s.pathProvider.GetRuntimeNamespace(), fileName, common.Sites)
+		if err != nil {
+			if opts.LogWarning {
+				fmt.Println("Site not initialized yet")
```

Comment:
I believe we should write this warning message to stderr.
The reason is that if you do `skupper site status my-site-name -o yaml > site.yaml` the warning message is included into the output file.

Reply:
changed to stderr